docs/reference/api.md

```
@@ -395,6 +395,172 @@ Removes cached images not referenced by config or any existing shed.
 
 The `deleted` array contains the images that were removed (or would be removed if `dry_run=true`).
 
+## System
+
+### GET /api/system/df
+
+Returns disk usage information for the server: image cache, per-instance rootfs copies and console logs, kernel/initrd, and orphan sidecar files.
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)